label_studio/tasks/functions.py

```diff
@@ -119,14 +120,15 @@ def export_project(project_id, export_format, path, serializer_context=None):
         tasks += ExportDataSerializer(_task_ids, many=True, **serializer_options).data
 
     # convert to output format
-    export_stream, _, filename = DataExport.generate_export_file(
+    export_file, _, filename = DataExport.generate_export_file(
         project, tasks, export_format, settings.CONVERTER_DOWNLOAD_RESOURCES, {}
     )
 
     # write to file
     filepath = os.path.join(path, filename) if os.path.isdir(path) else path
-    with open(filepath, 'wb') as file:
-        file.write(export_stream.read())
+    with open(filepath, 'wb'):
+        shutil.copy2(export_file.name, filepath)
+    export_file.close()
```

Comment:
I can confirm that this part works:

```
~/label_studio$ label-studio export 20 yolo
...
{"asctime": "21/Mar/2024:00:35:09 +0000", "name": "label_studio.server", "funcName": "main", "lineno": 335, "levelname": "INFO", "user_id": null, "message": "Project exported successfully: /label-studio/data/export/project-20-at-2024-03-21-00-35-efa9bad9.zip", "request_id": null}
```

And the zip file contents looked good.